workers/notificationworker/notificationworker.py

```diff
@@ -48,15 +49,18 @@ def create_gunicorn_worker():
 
     utilizing this method will enforce a 1:1 quay worker to gunicorn worker ratio.
     """
+    from flask import Flask
+
     note_worker = NotificationWorker(
         notification_queue, poll_period_seconds=10, reservation_seconds=30, retry_after_seconds=30
     )
+    app = Flask(__name__)
```

Comment:
hmmm `create_gunicorn_worker` creates this flask app, and `from singletons.config import app_config` creates another one - do we really need both? furthermore, what kind of side effects could this cause? 🤔

Reply:
This is just to make gunicorn happy. On next iterations most likely I'll remove `app` from GunicornWorker and replace it with some basic stub like `start_response("200 OK"); return []`. We don't expect it to serve any HTTP requests anyway, we only want it to watch code changes in local-dev.